pytests/test_dwts.py

```diff
@@ -3,11 +3,20 @@
 from numpy.testing import assert_array_almost_equal
 from scipy.sparse.linalg import lsqr
 
-from pylops.signalprocessing import DWT, DWT2D
+from pylops.signalprocessing import DWT, DWT2D, DWTND
 from pylops.utils import dottest
 
 par1 = {"ny": 7, "nx": 9, "nt": 10, "imag": 0, "dtype": "float32"}  # real
 par2 = {"ny": 7, "nx": 9, "nt": 10, "imag": 1j, "dtype": "complex64"}  # complex
+par3 = {"ny": 7, "nx": 9, "nz": 9, "nt": 10, "imag": 0, "dtype": "float32"}  # real 4D
```

Comment:
change comment from real 4d to complex 4d as you use imag=1j

Reply:
I might be missing something here but `par3` should be using `imag=0` and only `par4` should be complex with `imag=1j`, right?

Reply:
Ops sorry, I guess I read the wrong line ;) ignore this comment